tests/test_simulator.py

```diff
@@ -100,3 +100,40 @@ def test_renaissance_man_memoing(
     student_with_memo.valuation.reset()
 
     assert len(student_with_memo.valuation._value_memo) == 0
+
+
+def test_sub_student(
+    renaissance3: RenaissanceMan,
+    schedule: list[ScheduleItem],
+    course: Course,
+):
+    bundle = [item for item in schedule if item.values[0] == "301"]
+
+    reduced_schedule = sub_schedule([bundle])
+
+    course_strings = sorted([item.values[0] for item in reduced_schedule])
+
+    course, slot, weekday, _ = reduced_schedule[0].features
```

Comment:
Instead of doing this, just include the slot and weekday fixtures in your function definition (you already have course)